_rules/SC1-3-1-heading-name-does-not-only-contain-spaces.md

```diff
@@ -0,0 +1,190 @@
+---
```

Comment:
filename needs to change to have unique id as a part of it & lose the sc number

Reply:
Fixed